Source/WebInspectorUI/UserInterface/Base/Utilities.js

```diff
@@ -633,7 +633,7 @@ Object.defineProperty(Array, "isTypedArray",
             || constructor === Uint8ClampedArray
             || constructor === Uint16Array
             || constructor === Uint32Array
-            || constructor === Float16Array
+            || (typeof Float16Array === "function" && constructor === Float16Array)
```

Comment:
we typically just check `window.Float16Array`

Reply:
fair point, this will have the same effect and will more concise